keadm/cmd/keadm/app/cmd/edge/join.go

```diff
@@ -288,6 +291,7 @@ func createEdgeConfigFiles(opt *common.JoinOptions) error {
 	if errs := validation.ValidateEdgeCoreConfiguration(edgeCoreConfig); len(errs) > 0 {
 		return errors.New(pkgutil.SpliceErrors(errs.ToAggregate().Errors()))
 	}
+	configFilePath := filepath.Join(util.KubeEdgePath, "config/edgecore.yaml")
```

Comment:
Would it be better to update the file in the user config path and make this file the component edgecore configuration file?
It will be tricky if there are two config files.

Reply:
fixed

Reply:
Will component `edgecore` startup with the specified config file?

Reply:
we need to add option `--config /path/to/config.yaml` to edgecore start command  like `edgecore --config /root/config.yaml`